js/foundation.orbit.js

```diff
@@ -349,13 +349,15 @@ class Orbit extends Plugin {
           this.options[`animInFrom${dirIn}`],
           function(){
             $newSlide.css({'display': 'block'}).attr('aria-live', 'polite');
+            _this.$element.trigger('endcurrentslidechange.zf.orbit', [$curSlide]);
```

Comment:
These should be documented, see the current docs, `@fires` and the comments that we use.